services/crawler/http.go

```diff
@@ -0,0 +1,55 @@
+package crawler
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+
+	"github.com/junan/fake-useragent"
+)
+
+func GetRequest(url string) ([]byte, error) {
+	client := &http.Client{}
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return []byte{}, nil
+	}
+
+	req.Header.Set("User-Agent", randomUserAgent())
+	res, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	// Closing response body
+	defer res.Body.Close()
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return []byte{}, err
```

Comment:
```suggestion
		return nil, err
```

To be consistent with the return above.